packages/backend/src/core/sync/LiveL2TransactionDownloader.test.ts

```diff
@@ -86,17 +87,19 @@ describe(LiveL2TransactionDownloader.name, () => {
 
       await waitForExpect(() => {
         firstTxs.forEach((tx, i) => {
-          expect(
-            mockL2TransactionRepository.addLiveTransaction
-          ).toHaveBeenNthCalledWith(
-            i + 1,
-            {
-              transactionId: tx.transactionId,
-              timestamp: tx.timestamp,
-              data: tx.transaction,
-            },
-            mockKnexTransaction
-          )
+          if (tx.parsedSuccessfully) {
```

Comment:
I'd suggest adding a test that checks what happens if the transaction is not parsed successfully.

Reply:
Added